omod/src/main/webapp/resources/js/screen/bill.js

```diff
@@ -194,10 +194,16 @@ curl(
 					break;
 				case BillStatus.POSTED:
 				case BillStatus.PAID:
-                    $saveButton.val(__("Adjust Bill"));
-                    $saveButton.click(this.billView.handleAdjustBill);
+                    var $showAdjustmentButton=$('#showBillAdjustmentButton');
```

Comment:
I'm confused how this code works. It seems like if the save button is hidden the adjustment button is not displayed but if the 'showAdjustmentButton' value is false the Adjust Bill button displayed. Have you tested this?